Core/RoverControllerPresets/ManipulatorControllers/SingleAxisManipulatorController.cs

```diff
@@ -12,16 +12,24 @@ public ManipulatorControl CalculateMoveVector()
 		if (Mathf.Abs(velocity) < LocalSettings.Singleton.Joystick.Deadzone) 
 			velocity = 0f;
 
-		ManipulatorControl manipulatorControl = new()
+		ManipulatorControl manipulatorControl = null;
```

Comment:
null is redundant needed here
```suggestion
		ManipulatorControl manipulatorControl;
```